### PR TITLE
Webhook openeye

### PR DESCRIPTION
**Pull request by ttayson**
Initial version of the script

Features:
> receives data from Stream at endpoint "/"
> Converts timestamp to Openeye API default
> Sends Alert request to openeye API

### CODE REVIEW

File: webhooks/webhook_alerts_OpenEye.py [hunk 1]
```diff
@@ -0,0 +1,104 @@
+"""
+# Install flask
```
Comment (marcbelmont) [inline webhooks/webhook_alerts_OpenEye.py:2]: move this to README.md

File: webhooks/webhook_alerts_OpenEye.py [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+app = Flask(__name__)
+
+
+@app.route("/", methods=["GET", "POST"])
+def handle_event():
+    if request.method == "GET":
+        return jsonify({"error": "Method not allowed"}), 405
+    else:
+        def convert_to_timestamp_microseconds(time_string):
```
Comment (marcbelmont) [inline webhooks/webhook_alerts_OpenEye.py:39]: define this function at the root level (no indentation)

File: webhooks/webhook_alerts_OpenEye.py [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(
+        description="Webhook server to receive JSON on /note route.")
+    parser.add_argument("--host", type=str, default="127.0.0.1",
+                        help="The host IP address to bind the server to.")
+    parser.add_argument("--port", type=int, default=5000,
+                        help="The port number to bind the server to. (Optional - Default = 5000)")
+    parser.add_argument("--debug", type=int, default=0,
```
Comment (marcbelmont) [inline webhooks/webhook_alerts_OpenEye.py:97]: you can use `action='store_true'` instead.

File: webhooks/webhook_alerts_OpenEye.py [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+                }
+            ]
+        })
+
+        headers = {
+            'Authorization': '!EXT-INTEGRATOR AKI={},AKS={}'.format(args.aki_token, args.aks_token),
+            'Content-Type': 'application/json',
+        }
+
+        response = requests.request("POST", url, headers=headers, data=payload)
```
Comment (marcbelmont) [inline webhooks/webhook_alerts_OpenEye.py:79]: handle exceptions: https://stackoverflow.com/questions/16511337/correct-way-to-try-except-using-python-requests-module

File: webhooks/webhook_alerts_OpenEye.py [hunk 5]
```diff
@@ -0,0 +1,106 @@
+"""
+# Install flask
```
Comment (marcbelmont) [inline webhooks/webhook_alerts_OpenEye.py:2]: remove this text and add a link to the readme instead.

File: webhooks/webhook_alerts_OpenEye.py [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+    parser.add_argument("--port", type=int, default=5000,
+                        help="The port number to bind the server to. (Optional - Default = 5000)")
+    parser.add_argument("--debug", action="store_true",
+                        help="Turn on Flask debug mode. (Optional)")
+    parser.add_argument("--aki_token", type=str, required=True,
+                        help="To get the AKI, in OWS you would go into Management > Integrations > Event Receiver API (Analytics) then choose API Access Keys.")
+    parser.add_argument("--aks_token", type=str, required=True,
+                        help="To get the AKS, in OWS you would go into Management > I\ntegrations > Event Receiver API (Analytics) then choose API Access Keys, The AKS is normally only shown one time, when you first generate the key.")
+    args = parser.parse_args()
+    app.run(host=args.host, port=args.port, debug=bool(args.debug))
```
Comment (marcbelmont) [inline webhooks/webhook_alerts_OpenEye.py:87]: nit: bool() is not needed.

File: webhooks/README.md [hunk 7]
```diff
@@ -53,10 +62,61 @@ Build Solution and Run WebhookReader.exe as a Console Application
...
+Required parameters:
+
+- --aki_token
+- --aks_token
+
+For external access
+
+- --host=0.0.0.0
+
+To perform the integration realize as etapas abaixo:
```
Comment (marcbelmont) [inline webhooks/README.md:117]:
```suggestion
Next, configure and start Stream:
```

File: webhooks/README.md [hunk 8]
```diff
@@ -53,10 +62,61 @@ Build Solution and Run WebhookReader.exe as a Console Application
...
+
+For external access
+
+- --host=0.0.0.0
+
+Next, configure and start Stream:
+
+- Set the [Camera-ID](https://guides.platerecognizer.com/docs/stream/configuration#hierarchical-configuration), present in the config.ini configuration file, equal to the Camera External_ID parameter provided by OpenEye.
+- Set the parameter webhook_targets in config.ini to the host and port of your webhook.
+
```
Comment (marcbelmont) [inline webhooks/README.md:121]: for clarity, it would be good to have an example with a snippet of the config.

Comment (ttayson) [reply webhooks/README.md:121]: i wil do it

**Review by marcbelmont — APPROVED** [review]

**Pull request merged**